docs/openapi/components/schemas/credentials/FoodGradeInspectionCertificate.yml

```diff
@@ -245,7 +261,7 @@ example: |-
             "Organization"
           ],
           "name": "Ace Foodstuffs",
-          "description": "Agricultural goods shipping & distribution",
+          "description": "Ag goods shipping & distribution",
```

Comment:
I think this is a step in the wrong direction. "Ag" is not as clear as "Agriculture".

Reply:
Agree